README.md

```diff
@@ -74,44 +74,26 @@ The following chapter describes how we have technically implemented such user st
   ```bash
   cd spring-boot
   ```
-- Execute the following `go` program locally and optionally pass as parameters:
+- Initialize your application, optionally passing the following parameters:
   - `-k | --kubeconfig` : /PATH/TO/KUBE/CONFIG
-  - `-n | --namespace` : openshift's project
+  - `-n | --namespace` : OpenShift project
+  - `-a | --application` : application name
 
   ```bash
-  sb init
+  sb init -a spring-boot-http
   ```
+  Here, we initialized a `spring-boot-http` application. This name will be used to create OpenShift resources and set up the environment.
 
-- Verify if the `initContainer` has been injected within the `DeploymentConfig`
+- Verify if the `initContainer` has been injected within the `spring-boot-http` (the name you chose for your application) `DeploymentConfig`
 
   ```bash
-  oc get dc/spring-boot-supervisord -o json | jq '.spec.template.spec.initContainers[0]'
-  
-  {
-    "env": [
-      {
-        "name": "CMDS",
-        "value": "echo:/var/lib/supervisord/conf/echo.sh;run-java:/usr/local/s2i/run;compile-java:/usr/local/s2i/assemble;build:/deployments/buildapp"
-      }
-    ],
-    "image": "quay.io/snowdrop/supervisord@sha256:0c6ad373a3aa991edcb9b5806aecd0d57467f74018c96c6299adb9a10aaa86da",
-    "imagePullPolicy": "Always",
-    "name": "copy-supervisord",
-    "resources": {},
-    "terminationMessagePath": "/dev/termination-log",
-    "terminationMessagePolicy": "File",
-    "volumeMounts": [
-      {
-        "mountPath": "/var/lib/supervisord",
-        "name": "shared-data"
-      }
-    ]
-  }
+  oc get dc/spring-boot-http -o jsonpath='{.spec.template.spec.initContainers[0].name}{"\n"}'
```

Comment:
Command using `jq` was added by Giorgios. So, if you prefer to use jsonpath instead of jq, I can accept but then the prereq section should be updated. Can you check with Giorgios this point please ?

Reply:
I don't think he will mind and this way, no need to depend on an external tool, right, @geoand?

Reply:
If it can be done with jsonpath, then I am of course in favor of it.

I always default to using jq because it's more powerful, but I'm definitely in favor of using built in stuff when possible